docs/develop/accessibility/accessibility-screen-reader.md

```diff
@@ -0,0 +1,135 @@
+* After an interaction on a component, like a search bar or a slider, is the result or new value announced?
+* Is the component state is announced? For example, a button being disabled or a menu expanded must be announced.
+
+## Make your elements accessible to screen readers
+
+Several [ARIA](http://www.w3.org/TR/wai-aria-1.1/) markup extensions are available to make your elements accessible. Below are some of the most useful ones. This is not an exhaustive list, many other attributes exist.
+
+### Role
+
+The ``role`` attribute is used to say what kind of component the user is focusing on. The roles are usually defined with a ``div`` element. The ``button`` and ``section`` elements implicitly have a ``role`` in HTML5.
+
+There are a lot of available **[roles](http://www.w3.org/TR/wai-aria/roles)**. Some of them are associated with other ARIA attributes. For example, a ``treeitem`` allows us to also use the ``aria-expanded`` attribute that defines whether the item is expanded or collapsed. A complete list of roles and their associated ARIA attribute can be found in this role [schema](http://www.w3.org/TR/wai-aria/rdf_model.png).
+
+#### Example
+
+```html
+<div role="button" onclick="foo()">Click here</div>
+<button onclick="foo()">Click here</button>
+```
+
+### Aria-label and Aria-labelledby
+
+``Aria-label`` and ``Aria-labelledby`` set some alternative content or title to an element. This is different from a description (see [Aria-describedby](#aria-describedby)).
+
+There are two ways to define a label.
+
+* ``Aria-label`` is a simple attribute to directly set the text that is read by the assistive technology.
+* ``Aria-labelledby`` makes a reference to an **ID** of a **label** element.
+
+**NOTE:** Use only one label attribute, never use them both in an element.
+
+#### Example
+
+With **Aria-label**:
+
+```html
+<div role="button" aria-label="Close"><i icon="closeIcon"></i></div>
+```
+
+With **Aria-labelledby**:
+
+```html
+<label id="closeLabel" class="accessible-hidden">Close</label>
+<div role="button" aria-labelledby="closeLabel"><i icon="closeIcon"></i></div>
+```
+
+The ``accessible-hidden`` class is part of PXT and can be used to make a label hidden on the screen but still visible for a screen reader (therefore, it will read it).
+
+### Aria-describedby
+
+This attribute is used to include an additional description of the element. It is usually read a **few seconds after** the ``aria-label`` and the ``role`` by the screen readers.
+
+#### Example
+
+```html
+<label id="closeDescription" class="accessible-hidden">Close the dialog and returns to the menu</label>
+<div role="button" aria-label="Close" aria-describedby="closeDescription"><i icon="closeIcon"></i></div>
+```
+
+The ``accessible-hidden`` class is part of PXT and can be used to make a label hidden on the screen but still visible for a screen reader (therefore, it will read it).
```

Comment:
Here again....

Reply:
Yes, the reason why is that I use a specific case in those two examples, and I was thinking that in case if the reader read just the second example (because it's what he's looking for) he would probably not understand this specific case without the explanation. What do you think about, would you prefer I remove this?